doc/cephfs/scrub.rst

```diff
@@ -131,3 +131,15 @@ Control (ongoing) File System Scrubs
    {
        "return_code": 0
    }
+
+Damages
+=======
+
+The damages that can be reported and repaired by File System Scrub are:
+
+* DENTRY
+
+* DIR_FRAG
+
+* BACKTRACE
```

Comment:
Should we add more details about each type of damage?

Reply:
PTAL

Reply:
LGTM. Can this be backported to Quincy and to Pacific?